arches/app/models/graph.py

```diff
@@ -50,6 +50,7 @@ class Graph(models.GraphModel):
     """
 
     objects = GraphQuerySet.as_manager()
+    saved_spatial_view_geometry_nodes = None
```

Comment:
I don't think this is going to work as a place to stash state, as a typical deployment uses multiple workers.